src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -282,12 +289,18 @@ val qualityFormat: Option[FastqQualityFormat] = None,
   validate(inputs.length == readStructures.length, "The same number of read structures should be given as FASTQs.")
   validate(readStructures.flatMap(_.sampleBarcodeSegments).nonEmpty, s"No sample barcodes found in read structures: " + readStructures.map(_.toString).mkString(", "))
 
+
   private val pairedEnd = readStructures.count(_.templateSegments.nonEmpty) match {
     case 1 => false
     case 2 => true
     case n => invalid(s"Found $n read structures with template bases but expected 1 or 2.")
   }
 
+  if (illuminaStandards) {
```

Comment:
Playing devil's advocate, this isn't mentioned in the usage and seems overly mean.  Why can't I get single-end illumina fastqs?